src/AppLauncher/AppLauncher.tsx

```diff
@@ -164,6 +164,18 @@ function AppLauncher({
           );
         })}
       </Box>
+      <Divider sx={{ margin: theme => theme.spacing(0, 0, 2.5, 0) }} />
+      <Box sx={{ textAlign: "center" }}>
+        <Link
+          data-testid="virto-status-link"
```

Comment:
nice 👍

Reply:
Lets try and be consistent though. Dave [suggested](https://github.com/IPG-Automotive-UK/react-ui/pull/931#discussion_r1736095165) "component-item" so in this case "app-launcher-status-link"